pbsmetrics/config/metrics_test.go

```diff
@@ -86,6 +86,27 @@ func TestMultiMetricsEngine(t *testing.T) {
 		metricsEngine.RecordAdapterBidReceived(pubLabels, openrtb_ext.BidTypeBanner, true)
 		metricsEngine.RecordAdapterTime(pubLabels, time.Millisecond*20)
 	}
+	labels_blacklist := []pbsmetrics.Labels{
```

Comment:
Can you please change the `labels_blacklist` variable to be in camel case rather than snake case as per Go standards?

Reply:
LOL. I'm the worse when it comes to naming standards. Corrected